lib/index.js

```diff
@@ -353,6 +354,71 @@ const feedbackHandler = async function feedbackHandler(options) {
   logger.info('Thank you!');
 };
 
+async function triggerUpsertHandler(options) {
+  // We have to use the position number here because of lack of good support
+  // for nested commands with positional arguments
+  // consider replacing with `nested-yargs` npm package
+  const triggerName = options._[2];
+  const triggerOptions = {
+    source: {
+      uri: options.sourceUri,
+    },
+    target: {
+      funcName: options.targetFunc,
+    },
+  };
+  if (options.errorHandlerFunc) {
+    triggerOptions.errorHandler = {
+      funcName: options.errorHandlerFunc,
+    };
+  }
+  const result = await triggers.upsert(triggerName, triggerOptions);
+  logger.info(options.json ? JSON.stringify(result) : result.status);
+}
+
+async function triggerRemoveHandler(options) {
+  // options = {"_":["triggers","remove","triggerName"],"$0":"bn"}
+  // We have to use the position number here because of lack of good support
+  // for nested commands with positional arguments
+  // consider replacing with `nested-yargs` npm package
+  const triggerName = options._[2];
+  const result = await triggers.remove(triggerName);
+  logger.info(options.json ? JSON.stringify(result) : result.status);
+}
+
+async function triggerListHandler(options) {
+  const result = await triggers.list();
+  const rawData = (result.list || []).map(({ name, config, updatedAt }) => ({
```

Comment:
i wish this was typescript too :-(

Reply:
When would list not exist?

Reply:
it's not supposed to, still pretty risky leaving that out...

Reply:
It shouldn't be, we have type validation on all of our APIs, if we were using the generated interfaces, which we can't just yet, you would have seen that this is unnecessary.

Reply:
so... are you saying I should remove it?